spec/index.bs

```diff
@@ -12841,17 +12841,18 @@ attachments used by this encoder.
                 [=Device timeline=] steps:
 
                 1. [$Validate the encoder state$] of |this|. If it returns false, return.
+                1. Let |maxViewportRange| be |this|.{{GPUDevice/limits}}.{{GPUSupportedLimits/maxTextureDimension2D}} &times; `2`.
```

Comment:
Made me realize I think we have a typo in the correspondence doc where it says:
> `min(maxImageDimension2D, maxImageDimensionCube, maxFramebufferWidth, maxFramebufferHeight, maxViewportDimensions[0], maxViewportDimensions[1], viewportBoundsRange[0], -viewportBoundsRange[1])`

it should say
> `min(maxImageDimension2D, maxImageDimensionCube, maxFramebufferWidth, maxFramebufferHeight, maxViewportDimensions[0], maxViewportDimensions[1], -viewportBoundsRange[0], viewportBoundsRange[1])`

... but also there's no point in including `viewportBoundsRange` in that formula at all since it's guaranteed to be way bigger than `maxViewportDimensions`, right? (per https://github.com/gpuweb/gpuweb/issues/373#issuecomment-1756964198)
> `min(maxImageDimension2D, maxImageDimensionCube, maxFramebufferWidth, maxFramebufferHeight, maxViewportDimensions[0], maxViewportDimensions[1])`